bug: "Cannot find module 'http-cookie-agent/http'" on Docker image v1.18.0

### Describe the bug

Upgrading the Docker image to v1.18.0 returns `Error: Cannot find module 'http-cookie-agent/http'`.

### Steps to reproduce

1. Set up an instance on Docker image v1.17.0
2. Upgrade to v1.18.0
3. Restart the container

### Impact

The instance refuses to start.

### Additional information

Here is a log of the error:

```
Error: Cannot find module 'http-cookie-agent/http'
Require stack:
- /app/apps/websocket/wssServer.cjs
   at Function._resolveFilename (node:internal/modules/cjs/loader:1401:15)
   at defaultResolveImpl (node:internal/modules/cjs/loader:1057:19)
   at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1062:22)
   at Function._load (node:internal/modules/cjs/loader:1211:37)
   at TracingChannel.traceSync (node:diagnostics_channel:322:14)
   at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
   at Module.require (node:internal/modules/cjs/loader:1487:12)
   at require (node:internal/modules/helpers:135:16)
   at ../../node_modules/node-unifi/unifi.js (/app/apps/websocket/wssServer.cjs:184819:49)
   at __require (/app/apps/websocket/wssServer.cjs:12:51) {
 code: 'MODULE_NOT_FOUND',
 requireStack: [ '/app/apps/websocket/wssServer.cjs' ]
}
Node.js v22.15.0
```

### Version

1.18.0

### Installation method

Docker Run

### Browser

Firefox

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,6 +1,7 @@
 Dockerfile
 .dockerignore
 node_modules
+**/node_modules
 npm-debug.log
 README.md
 .next

--- a/apps/tasks/package.json
+++ b/apps/tasks/package.json
@@ -10,7 +10,7 @@
   "main": "./src/main.ts",
   "types": "./src/main.ts",
   "scripts": {
-    "build": "esbuild src/main.ts --bundle --platform=node --loader:.scss=text --external:@opentelemetry/api --external:http-cookie-agent --outfile=tasks.cjs",
+    "build": "esbuild src/main.ts --bundle --platform=node --loader:.scss=text --external:@opentelemetry/api --external:deasync --outfile=tasks.cjs",
     "clean": "rm -rf .turbo node_modules",
     "dev": "pnpm with-env tsx ./src/main.ts",
     "format": "prettier --check . --ignore-path ../../.gitignore",

--- a/apps/websocket/package.json
+++ b/apps/websocket/package.json
@@ -7,7 +7,7 @@
   "main": "./src/main.ts",
   "types": "./src/main.ts",
   "scripts": {
-    "build": "esbuild src/main.ts --bundle --platform=node --outfile=wssServer.cjs --external:bcrypt --external:@opentelemetry/api --external:http-cookie-agent --external:cpu-features --loader:.html=text --loader:.scss=text --loader:.node=text",
+    "build": "esbuild src/main.ts --bundle --platform=node --outfile=wssServer.cjs --external:bcrypt --external:@opentelemetry/api --external:deasync --external:cpu-features --loader:.html=text --loader:.scss=text --loader:.node=text",
     "clean": "rm -rf .turbo node_modules",
     "dev": "pnpm with-env tsx ./src/main.ts",
     "format": "prettier --check . --ignore-path ../../.gitignore",

